_sass/blocks/revenues/_revenues_subpage_steps.scss

```diff
@@ -20,21 +20,28 @@ $onshore-renewables-secondary-color: $blue;
 @import 'revenues_subpage_diving_graphics';
 
 .revenues_subpage-dyk {
-  position: relative;
   border-radius: $base-border-radius;
   margin-bottom: $base-padding;
   overflow: hidden;
+  position: relative;
   z-index: 3;
 
   @include respond-to(medium-up) {
     display: block;
     margin-top: 40%;
+
     @include span-columns(2);
+
     &[accordion-open=true] {
       @include span-columns(3);
     }
   }
 
+  border-radius: $base-border-radius;
```

Comment:
Expected item on line 40 to appear before line 29. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Property `border-radius` already defined on line 23